src/clj/medusa/changesets.clj

```diff
@@ -0,0 +1,82 @@
+(ns clj.medusa.changesets
+  (:require [clj-http.lite.client :as client]
+            [pl.danieljanus.tagsoup :as tagsoup]))
+
+(defn split-buildid [buildid]
```

Comment:
Private functions should be defined with defn-. Same goes for the other functions below.